source/includes/fact-minimum-fcv.rst

```diff
@@ -1,2 +0,0 @@
-The minimum supported server :ref:`feature compatibility version <view-fcv>` 
```

Comment:
double checking - are we referencing `/includes/fact-minimum-fcv.rst` anywhere else on our docs? If so, we should probs just link to MongoDB Server Version Compatibility page instead